--- a/lib/alpm_stats.sh	2016-03-06 11:46:54.000000000 +0100
+++ b/lib/alpm_stats.sh	2017-02-15 12:04:11.000000000 +0100
@@ -40,9 +40,9 @@ show_pkgs_stats() {
 	echo_fill "$CBLUE" - "$C0"
 	if [[ -t 1 ]]; then
 		printf "$CBLUE%${COLUMNS}s\r|$C0$CBOLD%*s $CGREEN%s$C0\n" \
-		    "|" $((COLUMNS/2)) "Archlinux " "($NAME $VERSION)"
+		    "|" $((COLUMNS/2)) "Manjaro Linux " "($NAME $VERSION)"
 	else
-		printf "%*s %s\n" 40 "Archlinux " "($NAME $VERSION)"
+		printf "%*s %s\n" 40 "Manjaro Linux " "($NAME $VERSION)"
 	fi
 	echo_fill "$CBLUE" - "$C0"
 	echo; echo_fill "$CBLUE" - "$C0"
