diff -Naur a/scripts/xflock4 b/scripts/xflock4
--- a/scripts/xflock4	2014-10-21 01:07:10.000000000 +1100
+++ b/scripts/xflock4	2014-10-27 19:12:06.957357152 +1100
@@ -27,7 +27,8 @@
 # Lock by xscreensaver or gnome-screensaver, if a respective daemon is running
 for lock_cmd in \
     "xscreensaver-command -lock" \
-    "gnome-screensaver-command --lock"
+    "gnome-screensaver-command --lock" \
+    "light-locker-command --lock"
 do
     $lock_cmd >/dev/null 2>&1 && exit
 done
@@ -35,7 +36,8 @@
 # else run another access locking utility, if installed
 for lock_cmd in \
   "xlock -mode blank" \
-  "slock"
+  "slock" \
+  "dm-tool lock"
   do
     set -- $lock_cmd
     if command -v -- $1 >/dev/null 2>&1; then
